content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Check whether Dockershim deprecation affects you
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
+
+The `dockershim` component of Kubernetes allows to use Docker as a Kubernetes's
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+Kubernetes' built-in `dockershim` component was deprecated in release v1.20.
+
+This page explains how your cluster could be using Docker as a
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}},
+provides details on the role that `dockershim` plays when in use, and shows steps
+you can take to check whether any workloads could be affected by `dockershim` deprecation.
+
+## Finding if your app has a dependencies on Docker
+
+If you are using Docker for building your application containers, you can still
+run these containers on any container runtime. This use of Docker does not count
+as a dependency on Docker as a container runtime.
+
+When alternative container runtime is used, executing Docker commands may either
+not work or yeild unexpected output. This is how you can find whether you have a
+dependency on Docker:
+
+1. Make sure no privileged Pods executing Docker commands.
+2. Check that scripts and apps running on nodes outside of Kubernetes
+   infrastructure do not execute docker commands. It might be:
+   - SSH to nodes to troubleshoot;
+   - Node startup scripts;
+   - Monitoring and security agents installed on nodes directly.
+3. Third-party tools that perform above mentioned privileged operations. See
+   [Migrating telemetry and security agents from
+   dockershim](migrating-telemetry-and-security-agents) for more information.
```

Comment:
please try avoid relative links ... the convention here is always have the English intrasite links start with "/docs"